docs/how-to/code/create-slice/task.yaml

```diff
@@ -0,0 +1,27 @@
+###########################################
+# IMPORTANT
+# Comments matter!
+# The docs use the wrapping comments as 
+# markers for including said instructions 
+# as snippets in the docs.
+###########################################
+summary: test the "Create a package slice for Chisel" guide
+
+execute: |
+  # [docs:apt-show-openssl]
+  apt show openssl
+  # [docs:apt-show-openssl-end]
+
+  git clone -b fix-copyright-symlink https://github.com/woky/chisel
```

Comment:
:/ pity

Reply:
what do you think of pining the Chisel commit to an early (pre-issue) one, like we do for the [rockcraft snap](https://github.com/canonical/rockcraft/blob/main/snap/snapcraft.yaml#L107) itself?